pages/levels/5.vue

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+const code = `    <div onClick="switchButtonColor()">Click me!</div>`;
+
+export default defineComponent({
+  data: () => {
+    return {
+      markdown: `What’s the deal with POSH? POSH stands for 'Plain Old Semantic HTML'. It basically means you should use the right HTML elements for their intended purpose, like using a button element for a clickable button instead of a div element.
```

Comment:
```suggestion
      markdown: `What’s the deal with POSH? POSH stands for 'Plain Old Semantic HTML'. It basically means you should use the right HTML elements for their intended purpose, like using a `button` element for a clickable button instead of a `div` element.
```